src/main/kotlin/org/rust/cargo/toolchain/tools/Cargo.kt

```diff
@@ -417,7 +417,7 @@ class Cargo(toolchain: RsToolchainBase, useWrapper: Boolean = false)
 
     fun checkNeedInstallCargoGenerate(): Boolean {
         val crateName = "cargo-generate"
-        val minVersion = SemVer("v0.9.0", 0, 9, 0)
+        val minVersion = SemVer.parseFromText("0.9.0")
```

Comment:
nit: maybe introduce some method, something like `parseSemVer` to verify that `SemVer.parseFromText` return non-null value? Currently, it's quite easy to make a mistake in the version string and nothing says you that you mistyped. As a result, run with coverage may not work because a user will have an outdated `grcov` version. The same with other tools

Reply:
Done